string jig
string guide

---------------
bool success
jig_guide_data data
